src/components/ChatWindow/ChatInput/ChatInput.tsx

```diff
@@ -70,20 +73,20 @@ export default function InputField({ conversation }: InputFieldProps) {
         rowsMin={1}
         rowsMax={3}
         className={classes.textArea}
-        aria-label="maximum height"
+        aria-label="chat input"
         placeholder="Write a message..."
         autoFocus
-        onKeyPress={isMobile ? () => null : event => handleReturnKeyPress(event)}
-        onChange={event => handleChange(event)}
+        onKeyPress={handleReturnKeyPress}
+        onChange={handleChange}
         value={messageBody}
       />
 
-      <div
+      <button
         className={clsx(classes.sendButton, { [classes.activeSendButton]: isValidMessage })}
         onClick={() => handleSendMessage(messageBody)}
       >
         <SendMessageIcon />
-      </div>
+      </button>
```

Comment:
Looks good! But now that this is a button, I think we should use the `disabled` prop when the button isn't supposed to be clickable:

```
      <button className={classes.sendButton} onClick={() => handleSendMessage(messageBody)} disabled={!isValidMessage}>
        <SendMessageIcon />
      </button>
```
When `disabled` is true, it tells the browser that the button isn't supposed to be clickable. As a result, the button won't be tabbable when it's disabled. So I think it goes a little further than just changing the styles so that it doesn't look clickable. 

Also, with this change, we can take advantage of the `:disabled` [pseudo-class](https://developer.mozilla.org/en-US/docs/Web/CSS/:disabled) to apply the right styles when it's disabled. This way we don't have to worry about conditionally applying the class ourselves. So the `sendButton` class can look like:

```
  sendButton: {
    height: '40px',
    width: '40px',
    border: '0',
    borderRadius: '4px',
    float: 'right',
    display: 'flex',
    alignItems: 'center',
    justifyContent: 'center',
    marginTop: '1em',
    background: '#0263E0',
    cursor: 'pointer',
    '&:disabled': {
      background: 'none',
      cursor: 'default',
    },
  },
  ```

The `&` is just a substitute for the classname of the parent object. So `'&:disabled'` gets turned into `.sendButton:disabled`.

Then, the test will have to be updated to look for the `disabled` prop instead of the `activeSendButton` class (which we can get rid of). For example:

```
    expect(
      wrapper
        .find(SendMessageIcon)
        .parent()
        .prop('disabled')
    ).toBe(true);
```

Reply:
good point! this looks nicer too